vaadin-list-box-flow-parent/vaadin-list-box-flow-integration-tests/src/test/java/com/vaadin/flow/component/listbox/test/ListBoxDataViewIT.java

```diff
@@ -223,28 +227,58 @@ public void testListDataView_nextAndPreviousItem_shouldComplyWithOriginalOrder()
     @Test
     public void testListDataView_setSortComparator_shouldSortTheItems() {
 
-        WebElement group = findElement(By.id(LIST_BOX_FOR_SORT_DATA_VIEW));
-        List<WebElement> buttons = group
+        WebElement listBox = findElement(By.id(LIST_BOX_FOR_SORT_DATA_VIEW));
+        List<WebElement> items = listBox
                 .findElements(By.tagName(VAADIN_ITEM));
 
         Assert.assertEquals("first rendered radio button's text should be",
-                "third", buttons.get(0).getText());
+                "third", items.get(0).getText());
         Assert.assertEquals("second rendered radio button's text should be",
-                "first", buttons.get(1).getText());
+                "first", items.get(1).getText());
         Assert.assertEquals("third rendered radio button's text should be",
-                "second", buttons.get(2).getText());
+                "second", items.get(2).getText());
 
         findElement(By.id(LIST_DATA_VIEW_SORT_BUTTON)).click();
         waitForElementPresent(By.tagName(VAADIN_ITEM));
 
-        group = findElement(By.id(LIST_BOX_FOR_SORT_DATA_VIEW));
-        buttons = group.findElements(By.tagName(VAADIN_ITEM));
+        listBox = findElement(By.id(LIST_BOX_FOR_SORT_DATA_VIEW));
+        items = listBox.findElements(By.tagName(VAADIN_ITEM));
 
         Assert.assertEquals("first rendered radio button's text should be",
-                "first", buttons.get(0).getText());
+                "first", items.get(0).getText());
         Assert.assertEquals("second rendered radio button's text should be",
-                "second", buttons.get(1).getText());
+                "second", items.get(1).getText());
         Assert.assertEquals("third rendered radio button's text should be",
-                "third", buttons.get(2).getText());
+                "third", items.get(2).getText());
+    }
+
+    @Test
+    public void setIdentifierProvider_forMultiSelectListBox_shouldSelectCorrectItems_dueToIdentifier() {
```

Comment:
Rework the naming https://sites.google.com/a/vaadin.com/fw-flow-team-s-wiki/contributing-code/flow-way-of-devloping